variables.tf

```
@@ -307,6 +307,7 @@ variable "node_pools" {
     node_count_min_per_zone    = number
     node_count_max_per_zone    = number
     node_labels                = map(string)
+    node_resource_labels       = optional(map(string), {})
```

Add documentation for node_resource_labels in the description of node_pools.

Can be like this:
```
  node_resource_labels: The GCE resource labels (a map of key/value pairs) to be applied to the nodes.
  Resource labels are applied to all nodes and persistent disks in the node pool. It can be used to track information
  about billing and usage. GKE automatically adds several resource labels to node pools. They should not be modified or deleted.
  See https://cloud.google.com/kubernetes-engine/docs/how-to/creating-managing-labels#automatically-applied-labels
```

Move `node_resource_labels` up to keep `node_labels` and `node_taints` together as they are related to each other.
```suggestion
    node_resource_labels       = optional(map(string), {})
    node_labels                = map(string)
```